lib/plugins/aws/provider/awsProvider.js

```diff
@@ -805,6 +806,19 @@ class AwsProvider {
     throw new Error(`Unrecognized function address ${functionAddress}`);
   }
 
+  resolveLayerArtifactName(layerName) {
+    const serverlessLayerObject = this.serverless.service.getLayer(layerName);
+    const layerArtifactName =
+      serverlessLayerObject.package && serverlessLayerObject.package.artifact
+        ? serverlessLayerObject.package.artifact
+        : path.join(
+            this.serverless.config.servicePath,
+            '.serverless',
+            this.provider.naming.getLayerArtifactName(layerName)
+          );
+    return layerArtifactName;
```

Comment:
I believe we can return resolved value directly (there seems no value in creating `layerArtifactName` variable)